mkdocs_simple_plugin/plugin.py

```diff
@@ -36,11 +162,17 @@ class SimplePlugin(BasePlugin):
     """
     config_scheme = (
         ('include_folders', config_options.Type(list, default=['*'])),
-        ('ignore_folders', config_options.Type(list, default=[])),
+        ('ignore_folders', config_options.Type(list, default=['site'])),
```

Comment:
Ah, I thought so, too. But unfortunately what I've found in my latest setup is that when I execute `mkdocs serve` it uses a tmp directory for site, so if there is a site directory lying around from a previous `mkdocs build` run, that site directory is *not* put in ignore_paths, and it gets searched for documentation, messing up the local server. So that's why I did this, but I would be happy for a cleverer solution if you have one.

Reply:
If by "bug" you mean "use a temp directory for site when doing mkdocs serve," that's standard behavior for mkdocs, see https://github.com/mkdocs/mkdocs/blob/master/mkdocs/commands/serve.py#L119. And as far as I know, standard behavior for "mkdocs build" is to leave the "site" dir behind in the project directory. Hence, you should find in your setup that "mkdocs build" followed by "mkdocs serve" is scanning the generated site for markdown files. Like this has simply gone under the radar since there generally aren't markdown files in the site directory. But you could force the issue by running "mkdocs build" and then hand-copying a markdown file into site/ and then running "mkdocs serve" and you should get a message about having a markdown file that's not mentioned in the navigation or something like that.

So anyhow, with more extensive extraction with this PR, it is definitely the case that it's more noticeable when a leftover site directory is being scanned under `mkdocs serve`. I do not think there is any "bug" involved. But should I file a separate issue for this point where we can discuss how you'd like to resolve it? I am happy to resolve it however you would like, but some action is needed for this PR to work with `mkdocs serve` in case there is a left-over site directory that should not be scanned. Thanks!

Reply:
The bug is that I expected the site_dir to be "site" within the folder and be ignored.  It looks like mkdocs serve is replacing it with a temp directory, which I wasn't aware of.

I think this looks like an ok solution for now.  I can think about the best way to get the original site_dir.